cli/command/manifest/fetch.go

```diff
@@ -0,0 +1,326 @@
+package manifest
```

Comment:
> That is effort and time that isn't necessary for the feature.

It is absolutely necessary for this to get merged.

>  Forcing the design to happen in the process here will lead to a substandard design

Why do you think the design phase is the wrong time to design the package? If the design is wrong, it can change, but making no decisions now is worse.

> Many packages in docker are exported and used in a single place, leading to a morass of exported interfaces without clear usage.

Exporting things allows for future reuse. If you don't export anything how do you expect someone to find the functionality they need? Code that is burred away in some unrelated package will never be found, and it will be duplicated.

Placing code in a package that fits its behaviour allows someone else to discover it and reuse it.  If an interface needs to change later, that's not a problem, but it first needs to be made available.